Catalyst and helper guides (#135)
Libera-Chat/libera-chat.github.io: content/_guides/catalyst.md

---
title: Catalysing and De-escalation
category: chanop
---

People often use Libera.Chat to get help with something on their computer.
Technical support channels are one of the things we're here for!

If you have ever had a technical problem, you know how overwhelming it can be.
Sometimes people seeking help will get very frustrated and might have a temper
or be rude. Patient and efficient help can be vital in avoiding conflict in
these situations.

This guide will outline some good practices for both operators and helpers for
interacting with people. It will also discuss some common situations and ideal
ways of responding.

> ## Only be "opped up" when necessary

Reviewer: I have critique on this part. This doesn't address services occassionally being down during which new ops cannot be reveived and doesn't communicate to people who don't already know how to get op attention.

https://mikaela.info/blog/english/2015/04/01/keep-the-ops-opped.html

Reviewer: Additionally Matrix users have additional difficulties seeing the op status of users as the current instance of the bridge only syncs ops to the other side and they need to use +O for syncing state as much as they can.

Reviewer 2: we try to warn users ahead of services downtime and people can (usually) check chanserv flags to find ops. the social benefit of not wearing far outweighs these points

Reviewer 2: (additionally, I don't want to make compromises on productively handling users for the sake of a matrix shortcoming)

Reviewer: Clients have a lot of differences in how they show globals or wallops, so they can be missed and it's possibly for ops to just not be on between the time of scheduled downtime announcement and beginning.

Reviewer 2: downtime is typically short, find network staff if you missed the wallops and have an urgent problem. it seems like a one in a million problem and certainly shouldn't discard a tried and true aspect of nonescalation

Reviewer: Sorry, I forgot in my previous comment, could there be a link somewhere on this section to another page explaining how to get connection to an operator whose flags aren't visible (including `/msg chanserv flags #channel` and how to connect e.g. account Mikaela to Aminda using information available through ChanServ or `/who`?

Reviewer 2: channels without viewable access lists typically don't want their ops found for whatever reason, I think it would be irresponsible to advise people on how to get around that desire

Reviewer: I am asking about channels with public access list. 

Using a real life example of a channel which has a compromise solution by only having a bot opped:

* There is spam on ##jrpg (and in this case I am pretending it's a much larger channel than it actually is)
* New IRC user doesn't know who are ops and how to reach them
* More advanced IRC user knows to /msg chanserv flags ##jrpg list
* The access list contains account name `Mikaela`. However currently that nickname doesn't exist on the channel. How will the user know that `Mikaela` is actually using name `Aminda`?

If there is no page explaining this issue, should I open a new issue to request one to be created which could then be linked from here?

Reviewer 3: We won't be able to handle every single potential edge case, and it is recommendations, not fixed rule. We think that the benefits of not being opped all the time are bigger than the issues and caveats, channel operators disagreeing are free to wear their op hat.  On finding channel ops, it has advantages and disadvantages, e.g. if there are three people wearing an op hat but all of them are afk, that is not helpful for people being new to IRC either, it is actually potentially frustrating. 

tl;dr: we know there are downsides to each variant, but we prefer this one, thus we put it in our recommendation and I think the wording is fine as is.

Reviewer 2: the point about instructions on querying `CS FLAGS` is a reasonable one though

Reviewer 3: I think it creates more edge cases (private versus public) and additional documentation needs (which flag is what, how do I reach someone who is not online, how do I find someone by account instead of nick) that will make it a lot harder to read, so if, then it should imho be split into separate documentation and linked.

Reviewer 2: yes, Mikaela suggested a separate document

Reviewer: @emersonveenstra (in https://github.com/Libera-Chat/libera-chat.github.io/issues/60#issuecomment-901961470) points out that it's documented in https://libera.chat/guides/faq#how-do-i-get-unbanned-or-unquieted so could that be linked somewhere here?

Author: As Fuchs points out above, we still need additional documentation which we'll then link to from various places. One specific page we need is a brief walk-through on advising what flags are actually needed for non-founder ops when setting up a new channel. One of the last pages i did, hurriedly with the intention of rewriting it before launch, was https://libera.chat/guides/creatingchannels which is a bit dire really and could do with explaining how to set up ops in it too. How to check your ops would be part of the checklist for that and probably an ideal place to link to from this.

Until that's done, we have already mentioned how to view flags on https://libera.chat/guides/faq#how-do-i-get-unbanned-or-unquieted but it's not an ideal place to throw people from this document in particular.

Author: Actually on second thoughts, a faq section of its own might be better, and have that faq section pointed to from the getting unbanned instructions. since pointing people seeking ops for a problem situation might not find it encouraging to be punted to instructions on how a banned person could undermine their efforts.

Most channels have a select few operators who are responsible for managing the
culture of the channel. These are the people with `+o` flags. Most clients
display a `@` next to their nick. Libera.Chat does not have "halfops" flags
like some other networks.

We recommend that operators in channels refrain from being "opped up" unless
it is necessary for immediate channel management activities like changing a

> locked topic or removing a problematic user. We discourage using the `+O`

Reviewer: I find the word "discourage" here as triggering negative reaction as I consider my critique on avoiding being opped as valid and I find it coming across as "no matter your reasoning to not follow this part, you must be a horrible person if you didn't follow this order".

Reviewer 2: you're projecting a lot of meaning on to "discourage"

Reviewer: I am aware. My request would be to rephrase this in a way that doesn't include a negation. E.g.

* remove this last phrase entirely.
* we recommend giving ops the AOP template.
* this can be achieved by using the AOP template.

Author: Discouraging isn't the same as forbidding. We don't forbid its use. You're not breaking any rules by doing it. You'd just be using (by our philosophy) non-best practices. Sometimes that might cause discomfort at doing something less than 100% correct, but that's not on us.

your community members feel like they're walking on eggshells. It is also
easier for bad actors to weaponize you against other community members.

If you are the only channel op, we encourage you to consider giving trusted
channel users ops as well. Not only does this provide you with the freedom to
do other things without worrying about the channel, but it also gives you
another person to talk with about situations that arise.

Find others in your community who are good at talking to people to help you
manage things. They don't need to be the most knowledgeable person in your
channel, but they need to be patient, fair, and approachable.

## Appeals processes and accountability

Removing people should ideally not be a permanent thing. People make mistakes
and have bad days. One mistake or bad day shouldn't be the end of their
participation in your channel.

Also, permanently excluding users from a channel can lead to them seeking a
way of evading into the channel, becoming even more problematic.

It's an excellent idea for your channel to have a way for people to appeal a
ban and negotiate being allowed back. Consider having an "ops" channel; most

> channels use #<channelname>-ops for this. Discourage non-ops from lingering

Reviewer 4: ```suggestion
channels use #\<channelname\>-ops for this. Discourage non-ops from lingering
```

> channels use `#<channelname>-ops` for this. Discourage non-ops from lingering
> in the channel, so people who approach you can do so with some privacy. When

Reviewer: ```suggestion
channels use `#<channelname>-ops` for this. Encourage non-ops to leave the channel once
their case has been handled, so people who approach you can do so with some privacy. When
```

I think this would be a more positive way to communicate the same thing.

Author: Well, now it's a bit specific to those who even had a case to start with and not also tourists. I'll come up with a compromise.

Author: Does the new wording i've committed address your concern?

Reviewer: Yes, thank you.

---
title: Catalysing and De-escalation
category: chanop
---

People often use Libera.Chat to get help with something on their computer.
Technical support channels are one of the things we're here for!

If you have ever had a technical problem, you know how overwhelming it can be.
Sometimes people seeking help will get very frustrated and might have a temper
or be rude. Patient and efficient help can be vital in avoiding conflict in
these situations.

This guide will outline some good practices for both operators and helpers for
interacting with people. It will also discuss some common situations and ideal
ways of responding.

## Only be "opped up" when necessary

Most channels have a select few operators who are responsible for managing the

> culture of the channel. These are the people with `+o` flags. Most clients

Reviewer 5: I wonder if we should make a clearer distinction between ChanServ flags and IRC modes, since there are often confusion about it. Maybe as simple as spelling out "people with the ChanServ flag +o" and later being voice (having mode +v) or similar

locked topic or removing a problematic user. We discourage using the `+O`
flag, which automatically "ops up" the flagged user on join. There are other
ways for [users to find out](faq#how-to-find-out-who-runs-a-channel) who runs
the channel.

Being opped up (or even voiced `+v` for manual or `+V` for automatic) in your
channel means you appear above normal users in your channel's user list, which
creates a visible hierarchy. While this might sound desirable, it can
interfere with your ability to give support. People will see you as an
authority figure and be wary of you. It's like the difference between having a
conversation with a boss or teacher, and having a conversation with a
co-worker or a classmate. Users may view your responses as orders, threats, or
trick questions when you do not intend them to be.

Channel Ops should avoid demonstrating hierarchy and interact as regular
participants in their channels as much as possible. Try not to be someone who
only appears when things are heated. Having a rapport with your users will
make it easier to de-escalate heated situations, making it easier to avoid
having to forcefully remove people from your community.

> ### Why is #libera different?

Reviewer 5: ```suggestion
### Why is `#libera` different?
```

Styleguide

Author: oh wasn't aware ticks were legal in headers or i'd have done that

> You may have noticed that the #libera channel doesn't follow this guideline.

Reviewer 5: ```suggestion
You may have noticed that the `#libera` channel doesn't follow this guideline.
```

Styleguide

> The #libera channel is relatively large and is a focal point for botnets and

Reviewer 5: ```suggestion
The `#libera` channel is relatively large and is a focal point for botnets and
```

Styleguide

> other nefarious behavior. It uses channel mode +z when dealing with unwanted

Reviewer 5: ```suggestion
other nefarious behavior. It uses channel mode `+z` when dealing with unwanted
```

Styleguide

messages, which requires staff to be opped, so they don't miss messages.

> Additionally, people often come to #libera to find network staff with

Reviewer 5: ```suggestion
Additionally, people often come to `#libera` to find network staff with
```

Styleguide

If you are the only channel op, we encourage you to consider giving trusted
channel users ops as well. Not only does this provide you with the freedom to
do other things without worrying about the channel, but it also gives you
another person to talk with about situations that arise.

Find others in your community who are good at talking to people to help you
manage things. They don't need to be the most knowledgeable person in your
channel, but they need to be patient, fair, and approachable.

## Appeals processes and accountability

Removing people should ideally not be a permanent thing. People make mistakes
and have bad days. One mistake or bad day shouldn't be the end of their
participation in your channel.

Also, permanently excluding users from a channel can lead to them seeking a
way of evading into the channel, becoming even more problematic.

It's an excellent idea for your channel to have a way for people to appeal a
ban and negotiate being allowed back. Consider having an "ops" channel; most
channels use `#<channelname>-ops` for this. Request that non-ops leave the
channel if they do not have an immediate issue, so that people can raise

> their issues with some privacy. When talking to banned users in an -ops

Reviewer 5: ```suggestion
their issues with some privacy. When talking to banned users in an `-ops`
```

I feel like this should be treated the same as a channel name since it's a partial channel name

channel, de-escalation tools and building rapport are critically important.

> If many channel operators speak to a person at once in an -ops channel, it can

Reviewer 5: ```suggestion
If many channel operators speak to a person at once in an `-ops` channel, it can
```

If nothing else just to make it more visually clear that the - isn't used for sentence pausing

seem like "ganging up on" the person. You may want to informally limit how
many ops are involved in such discussions at once.

Operator channels also allow people a place to report issues without
escalating situations in the channel. It is a good idea to advertise it in the
channel's topic. If people cannot get into your ops channel, they cannot use
it, so banning from that should be very rare.

> It is also a good idea to ensure that people can use ChanServ to find nicks of

Reviewer 5: ```suggestion
It is also a good idea to ensure that people can use `ChanServ` to find nicks of
```

Styleguide
